app/views/api_docs/vendor_api_docs/pages/home.md

```diff
@@ -75,21 +75,73 @@ Each token is associated with a single provider. It will grant access to applica
 
 For instructions on how to authenticate see the [API reference](/api-docs/reference#authentication).
 
-### Versioning
+## API versioning strategy
 
-The version of the API is specified in the URL `/api/v{n}/`. For example: `/api/v1/`, `/api/v2/`, `/api/v3/`, ...
+When we provide new features through the API, we sometimes make changes that require you to update student record systems.
 
-When the API changes in a way that is backwards-incompatible, a new version number of the API will be published.
+We give a new number to the new version of the API. The change of version number shows whether we’ve made breaking or non-breaking changes to the API.
 
-When a new version, for example `/api/v2`, is published, both the previous **v1** and the current **v2** versions will be supported.
+### Breaking changes
 
-We, however, only support one version back, so if the **v3** is published, the **v1** will be discontinued.
+Breaking changes usually involve modifying or deleting parts of the API, such as:
 
-When non-breaking changes are made to the API, this will not result in a version bump. An example of a non-breaking change could be the introduction of a new field without removing an existing field.
+- removing features
+- removing fields
+- changing the behaviour of endpoints, for example requiring a new parameter in order for applications to be synced
 
-Information about deprecations (for instance attributes/endpoints that will be modified/removed) will be included in the API response through a ‘Warning’ header.
+You must update student record systems before you move to a new version with breaking changes.
 
-We will update our [release notes](/api-docs/release-notes) with all breaking and non-breaking changes.
+### Non-breaking changes
+
+When we make non-breaking changes the API remains ‘backward compatible’. This means that the changes do not affect the existing functionality of the API.
+
+Non-breaking changes include adding new:
+
+- endpoints, for example to allow individual conditions to be marked as met or not met
+- nested resources and objects, for example details of interviews
+- fields, for example when candidates are asked a new question
+- optional query parameters, for example to allow optional pagination when applications are synced
+
+You do not need to update student record systems before moving to a new version with non-breaking changes. You only need to make updates if you want to use the version’s new features.
+
+### How the API version number reflects the changes we’ve made
+
+We use the format major.minor (for example, 1.2) to indicate the API version.
```

Comment:
could we wrap version numbers with `` ?

Reply:
when we've got `v1.1` should the v be included in the \`\`